src/rest-server/src/models/job.js

```diff
@@ -434,6 +434,10 @@ class Job {
   }
 
   generateDockerContainerScript(data, idx) {
+    let paiMachineList = '';
+    if (fs.existsSync('/pai-cluster-config/cluster-configuration.yaml')) {
+      paiMachineList = yaml.safeLoad(fs.readFileSync('/pai-cluster-config/cluster-configuration.yaml', 'utf8'))['machine-list']
```

Comment:
Use try-catch block around read instead of exists-read to prevent race conditions.